CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Don't include changes that are purely internal. The CHANGELOG should be a
   useful summary for people upgrading their application, not a replication
   of the commit log.
+- Interventions: allow any text and update design
```

Comment:
What are the lines above are, if they're neither a breaking or unreleased ?
Moved to an Unreleased section as suggested. It's not actually breaking because the component has been removed in the apps where it was used

Reply:
The lines in the first section under `#Changelog` are instructions. Nitpick - can you use a `*` at the start of the line, like the older entries? (line 12 is using a `+` but I think that's been corrected in `master` if you do a rebase).